workspace-manager-clienttests/README.md

```diff
@@ -0,0 +1,32 @@
+# workspace-manager-clienttests
+This Gradle project contains Test Runner tests written with the Workspace Manager Service client library.
+
+The Test Runner library [GitHub repository](https://github.com/DataBiosphere/terra-test-runner) has documentation for
+how to write and execute tests.
+
+#### Run a test
+The workspace manager tests require the appropriate service account keys to be available in the `rendered/` folder.
+
+Run the following command from the workspace-manager-clienttests directory to retrieve the required keys.
+Note: `render-config.sh` assumes a valid `.vault-token` in your `$HOME` directory.
```

Comment:
Worth adding a reference for how to generate `.vault-token`? https://github.com/broadinstitute/dsde-toolbox#authenticating-to-vault

Reply:
Added additional notes to PR on how to generate `vault-token`.